app/notifiers/course/assessment_notifier.rb

```diff
@@ -3,6 +3,20 @@ class Course::AssessmentNotifier < Notifier::Base
   def assessment_attempted(user, assessment)
     create_activity(actor: user, object: assessment, event: :attempted).
       notify(assessment.tab.category.course, :feed).
-      save
+      save!
+  end
+
+  # To be called when user submitted an assessment.
+  def assessment_submitted(user, course_user, submission)
+    # TODO: Replace with a group_manager method in course_user
+    managers = course_user.groups.includes(group_users: [course_user: [:user]]).
```

Comment:
`g.group_users.select(&:manager?)`, otherwise, the eager load will be overwritten.